assets/component-card.css

```diff
@@ -166,121 +61,214 @@
   padding-right: 1.2rem;
 }
 
-.card-information__wrapper {
+.card-information {
   width: 100%;
 }
 
-.card-information__wrapper > * {
+.card-information > * {
   line-height: calc(1 + 0.4 / var(--font-body-scale));
   color: rgb(var(--color-foreground));
 }
 
-.card-information__wrapper > .price {
+.card-information > .price {
   color: rgb(var(--color-foreground));
 }
 
-.card-information__wrapper > .rating {
+.card-information > .rating {
   margin-top: 0.4rem;
 }
 
-.card-information__wrapper
+.card-information
   > *:not(.visually-hidden:first-child)
   + *:not(.rating) {
   margin-top: 0.7rem;
 }
 
-.card-information__wrapper .caption {
+.card-information .caption {
   letter-spacing: 0.07rem;
 }
 
 .card-wrapper {
   color: inherit;
-  display: flex;
-  flex-direction: column;
+  height: 100%;
   position: relative;
   text-decoration: none;
   padding-bottom: 0.5rem;
 }
 
-.card-wrapper .card-information {
-  order: 1;
+.card__media-spacer {
+  padding: 2rem 2rem 0;
 }
 
-.card-wrapper .full-unstyled-link::after {
-  content: "";
+@media screen and (min-width: 750px) {
+  .card__media-spacer {
+    padding: 3rem 3rem 0;
+  }
+}
+
+.card-article-info {
+  margin-top: 1rem;
+}
+
+.overlay-card {
   position: absolute;
   top: 0;
-  right: 0;
   bottom: 0;
   left: 0;
+  right: 0;
+  display: flex;
+  background: transparent;
+  transition: background-color var(--duration-long) ease;
+}
+
+.card:hover .card-colored .overlay-card {
+  background-color: rgba(var(--color-card-hover), 0.06);
+}
+
+/* Update */ 
+
+.card {
+  text-align: var(--card-text-alignment);
+  text-decoration: none;
+}
+
+.card--card,
+.card--standard .card__inner { 
+  border-radius: var(--card-corner-radius);
+  border: var(--card-border-width) solid rgba(var(--color-foreground), var(--card-border-opacity));
```

Comment:
In this case, the border color is white as defined by the color-scheme. I've now prevented the background-color from being visible when the border opacity is zero. https://github.com/Shopify/dawn/pull/886/commits/cfe147f21458baf110fd97ce99f9afdf83cbd30b

Reply:
Im curious to hear designer's thoughts on this. 

I feel like maybe in the instances where Accent 1, Accent 2 and Inverse are used, we should use background color instead of the foreground/font color. This way you can still have the effect you expect to see as a user I think 🤔 
![](https://screenshot.click/21-11-t2l0n-95296.png)